scripts/start.sh

```diff
@@ -107,8 +107,19 @@ echo -e "\nStart consumers for additional topics: WIKIPEDIANOBOT, EN_WIKIPEDIA_G
 ${DIR}/consumers/listen_WIKIPEDIANOBOT.sh
 ${DIR}/consumers/listen_EN_WIKIPEDIA_GT_1_COUNTS.sh
 
-echo -e "\nWaiting for KSQL queries to start, sleeping 50 seconds"
-sleep 50
+# Verify wikipedia.parsed topic is populated and schema is registered
+MAX_WAIT=50
+CUR_WAIT=0
+echo -e "\nWaiting up to $MAX_WAIT seconds for wikipedia.parsed topic to be populated"
+while [[ ! $(docker-compose exec schemaregistry curl -X GET --cert /etc/kafka/secrets/schemaregistry.certificate.pem --key /etc/kafka/secrets/schemaregistry.key --tlsv1.2 --cacert /etc/kafka/secrets/snakeoil-ca-1.crt https://schemaregistry:8085/subjects) =~ "wikipedia.parsed-value" ]]; do
+  sleep 10
```

Comment:
why 10 sec (instead of say 1 sec) if you really want to minimize startup time?

Reply:
Agree and makes sense.  I have reduced it to 3 seconds.  (1 second is a bit much, given that waiting for response from Docker takes ~1 second.)